pages/blog/index.tsx

```diff
@@ -34,4 +36,40 @@ if (import.meta.main) {
     output(import.meta.url),
     html(renderToString(<Index />), { title: "Blog ▲ Nova" }),
   );
+
+  const feed = new Feed({
+    title: "Nova Blog",
+    description: "Nova Blog",
+    id: "https://trynova.dev/blog",
+    link: "https://trynova.dev/blog",
+    language: "en",
+    favicon: "https://trynova.dev/favicon.svg",
+    copyright: `Copyleft ${new Date().getFullYear()} The Nova Contributors`,
+  });
+
+  allPosts
+    .forEach((post) => {
+      // BASE_URL doesn't seem to be defined, even in the prodution build, so `href` sometimes just returns a path, so we have to handle that case
+      const possiblyPath = href(post.file);
+      const link = possiblyPath.startsWith('https://') ? possiblyPath : `https://trynova.dev${possiblyPath}`;
+
+      feed.addItem({
+        title: post.meta.title,
+        id: link,
+        link,
+        description: post.meta.description,
+        content: marked.parse(post.body, { async: false }) as string,
+        author: post.meta.authors.map((author) => ({
+          name: author.name,
+          link: author.url
+        })),
+        date: new Date(
+          post.meta.date.toZonedDateTime('Etc/Greenwich').epochMilliseconds
+        ),
```

Comment:
```suggestion
        date: new Date(
          post.meta.date.toZonedDateTime('UTC').epochMilliseconds
        ),
```
Let's be consistent and use UTC in case the TZ entries differ somehow.

Reply:
Etc/Greenwich is UTC +0:00. Are you sure that "UTC" works as a value here? I'm not familiar with the Temporal API.

Reply:
Yep, I'm sure: https://github.com/trynova/website/blob/main/pages/blog/utils.ts#L36

`GMT` and `UTC` should always be the same but seem to be defined as [different TZ db entries](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) so lets use `UTC`.